app/models/spotlight/blacklight_configuration.rb

```diff
@@ -164,7 +164,7 @@ def blacklight_config
           config.facet_fields = Hash[config.facet_fields.sort_by { |k, _v| field_weight(facet_fields, k) }]
 
           config.facet_fields.each do |k, v|
-            v.original = v.dup
+            v.original = v.dup unless v.custom_field
```

Comment:
Would it be possible to include a test for this behavior?

Reply:
Yeah, I can work on it this afternoon.